route-rs-runtime/src/lib.rs

```diff
@@ -3,7 +3,26 @@ extern crate futures;
 extern crate crossbeam;
 extern crate tokio;
 
+/// Elements are the unit of transformation in route-rs. The Element is defined by a trait that requires that
```

Comment:
This is an annoying requirement, but Github is clipping these comments because the lines are too long. Can you make them shorter? I guess `rustfmt` doesn't mangle comment lines?

Reply:
This is an unstable option so I guess we'll have to fix our comments manually for now.